### PR TITLE
Initial commit of System.Console tests

### PR DESCRIPTION
**Pull request by MattGal**

### CODE REVIEW

File: src/System.Console/tests/packages.config [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <package id="System.Collections" version="4.0.10-beta-22412" />
+  <package id="System.IO" version="4.0.10-beta-22412" />
+  <package id="System.IO.FileSystem" version="4.0.0-beta-22412" />
+  <package id="System.Runtime" version="4.0.20-beta-22412" />
+  <package id="System.Runtime.Extensions" version="4.0.10-beta-22412" />
+  <package id="System.Threading" version="4.0.0-beta-22412" />
+  <package id="System.Threading.Overlapped" version="4.0.0-beta-22412" />
+  <package id="System.Threading.Tasks" version="4.0.10-beta-22412" />
+  <package id="System.Threading.Thread" version="4.0.0-beta-22412" />
+  <package id="System.Threading.ThreadPool" version="4.0.10-beta-22412" />
```
Comment (weshaggard) [inline src/System.Console/tests/packages.config]: Are all these dependencies actually necessary? I didn't see any reference to ThreadPool for example.

File: src/System.Console/tests/System.Console.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{99E5069D-241F-48A6-8F29-404B4AED72BF}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AssemblyName>System.Console.Tests</AssemblyName>
+    <RootNamespace>System.Console.Tests</RootNamespace>
+  </PropertyGroup>
+
```
Comment (weshaggard) [inline src/System.Console/tests/System.Console.Tests.csproj]: Please add the VS configuration:

  
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
  </PropertyGroup>

**Comment by weshaggard** [comment]
This changes should go to v1.0 branch. and we need to make sure the CI system runs on it, not sure why that didn't kick in.

**Comment by ellismg** [comment]
Closing in favor of #650

**Pull request closed without merging**